superset-frontend/src/explore/components/controls/FilterControl/AdhocFilterEditPopoverSimpleTabContent/index.tsx

```diff
@@ -141,8 +141,6 @@ export const useSimpleTabFilterProps = (props: Props) => {
       );
     }
     return !(
-      (props.datasource.type === 'druid' &&
-        TABLE_ONLY_OPERATORS.indexOf(operator) >= 0) ||
       (props.datasource.type === 'table' &&
         DRUID_ONLY_OPERATORS.indexOf(operator) >= 0) ||
```

Comment:
what's this druid_only_operators?

Reply:
It leads to an enum that is Operators.Regex. But it only exists in this file, so I deleted it.